alerts.js

```diff
@@ -0,0 +1,14 @@
+
+// ruleid:javascript-prompt
+var name = prompt('what is your name');
+// ruleid: javascript-alert
+alert('your name is ' + name);
+alert('not', 'a', 'valid', 'alert')
+// ruleid: javascript-confirm
+if ( confirm("pushem!") == true) {
+    r = "x";
+} else {
+    r = "Y";
+    // ruleid: javascript-debugger
+    debugger;
```

Comment:
:exclamation:**Cycode**: SAST violation: 'Found debugger call'.
 
found debugger call; should this be in production code?
 
Severity: Medium
 <!-- 4f75ee0f-3c55-40ba-9fd4-876ac3ead28f -->